tools/pelican/content/ipynb/tutorial-accessing-cases-and-deaths.ipynb

```diff
@@ -0,0 +1,6219 @@
+       "L 211.6 82.056196 \n",
+       "L 213.616867 79.876081 \n",
+       "L 215.633735 76.78125 \n",
+       "L 219.66747 71.072243 \n",
+       "L 221.684337 68.581501 \n",
+       "L 223.701205 66.737354 \n",
+       "L 225.718072 64.506558 \n",
+       "L 227.73494 61.751782 \n",
+       "L 229.751807 59.588834 \n",
+       "L 231.768675 57.119345 \n",
+       "L 235.80241 53.292905 \n",
+       "L 237.819277 51.871375 \n",
+       "L 241.853012 48.456925 \n",
+       "L 247.903614 43.767266 \n",
+       "L 249.920482 42.631841 \n",
+       "L 255.971084 39.574612 \n",
+       "L 260.004819 37.344634 \n",
+       "L 262.021687 36.459345 \n",
+       "L 266.055422 35.195582 \n",
+       "L 268.072289 34.04953 \n",
+       "L 276.139759 31.152519 \n",
+       "L 278.156627 30.600747 \n",
+       "L 280.173494 30.232081 \n",
+       "L 282.190361 29.567502 \n",
+       "L 290.257831 27.419267 \n",
+       "L 292.274699 26.985206 \n",
+       "L 296.308434 26.41545 \n",
+       "L 302.359036 25.031522 \n",
+       "L 306.392771 24.419259 \n",
+       "L 310.426506 23.995825 \n",
+       "L 312.443373 23.733426 \n",
+       "L 314.460241 23.588739 \n",
+       "L 316.477108 23.165305 \n",
+       "L 342.696386 20.19963 \n",
+       "L 344.713253 20.320611 \n",
+       "L 350.763855 19.741045 \n",
+       "L 352.780723 19.648674 \n",
+       "L 354.79759 19.177011 \n",
+       "L 358.831325 18.726601 \n",
+       "L 379 17.207796 \n",
+       "L 379 17.207796 \n",
+       "\" style=\"fill:none;stroke:#e24a33;stroke-linecap:square;stroke-width:1.5;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_3\">\n",
+       "    <path d=\"M 44.2 224.76416 \n",
+       "L 44.2 7.32416 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_4\">\n",
+       "    <path d=\"M 379 224.76416 \n",
+       "L 379 7.32416 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_5\">\n",
+       "    <path d=\"M 44.2 224.76416 \n",
+       "L 379 224.76416 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "   <g id=\"patch_6\">\n",
+       "    <path d=\"M 44.2 7.32416 \n",
+       "L 379 7.32416 \n",
+       "\" style=\"fill:none;stroke:#ffffff;stroke-linecap:square;stroke-linejoin:miter;\"/>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"p20576e858e\">\n",
+       "   <rect height=\"217.44\" width=\"334.8\" x=\"44.2\" y=\"7.32416\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cases.plot()"
```

Comment:
adding a trailing semicolon, it will avoid printing text ```<matplotlib.axes._subplots.Axes...```

Reply:
Good point, done.